packages/@expo/cli/src/start/doctor/dependencies/validateDependenciesVersions.ts

```diff
@@ -96,6 +96,7 @@ export async function getVersionedDependenciesAsync(
   const combinedKnownPackages = await getCombinedKnownVersionsAsync({
     projectRoot,
     sdkVersion: exp.sdkVersion,
+    skipRemoteVersions: await isExpoPreReleaseAsync(projectRoot),
```

Comment:
It's common for us to log a warning in cases like this (e.g. unversioned warnings)

Reply:
Will add it.